src/main/java/com/neprozorro/model/LotStatus.java

```diff
@@ -0,0 +1,8 @@
+package com.neprozorro.model;
+
+public enum LotStatus {
+
+    CLARIFICATION_PERIOD, WAITING_FOR_ORDER, PREQUALIFICATION, PREQUALIFICATION_APPEAL_PERIOD, FIRST_INTERMEDIATE_STAGE,
```

Comment:
they should be in one style with other ENUMs in the project

Reply:
Decided to add Ukrainian descriptions

Reply:
Done